README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+# hw01
+## Autobiography
+My name is Olivia Padilla, and I am a third-year undergraduate studying Public Policy.  I am from a very small town in the rural Ozarks region of southwest Missouri. Here is a link to my town's webpsite. [link](http://www.cityofcassville.com/). In my free time I like to read, my favorite book is *The Phantom Tollbooth* by Norton Juster. I have three goals for this quarter: 
+1. Practice social distancing. 
+2. Maintain my mental health while doing so. 
+3. Plan an awesome cross-country road-trip for when this is all over.  
+
+## Process
```

Comment:
Not a big deal at all, but in future assignments it might be better if you leave the reflection to the pull request textbox. That way your report is clean.